doc/lsd.md

```diff
@@ -125,6 +125,9 @@ lsd is a ls command with a lot of pretty colours and some other stuff to enrich
 `-U`, `--no-sort`
 : Do not sort. List entries in directory order
 
+`-Z` `--label`
```

Comment:
Let's use `--context` as gnu ls does that.

Reply:
Can't believe I missed coreutils having a long option. Changed every occurence of label to context so code is consistent with CLI.